.codeclimate.yml

```diff
@@ -0,0 +1,19 @@
+version: "2"
+checks:
+  method-complexity:
+    enabled: true
+    config:
+      threshold: 5
+  file-lines:
+    enabled: true
+    config:
+      threshold: 250
+  method-lines:
+    enabled: true
+    config:
+      threshold: 50
+exclude_patterns:
```

Comment:
Just trying to understand this. What does this line do exactly?  Are you excluding the tests and coverage?

Reply:
Yes, these settings tell code-climate the files to analyze. Since the tests are written extensively and can contain duplicate codes that can't be refactored, they are excluded. 
The coverage files are auto generated when we run npm coverage and are only used to send the reports to coveralls. Since they are JS files, they can raise issue of bad code while we have no control.

All files that are not user typed, should be excluded from code analysis and that is the purpose of the block

Reply:
### I will ensure that next time.

Although, I have seen such cases while I haven't edited a thing in a file. Not in this case though because I was sending the reports directly to the code-climate through GitHub actions I had the same file and resolved the conflict during re-base. 
Screenshot attached below, maybe that was the case. 
![image](https://user-images.githubusercontent.com/28519395/154038340-9ab3e674-42c1-4b91-bbde-f4db53bf12e4.png)

And so many others where we have App.js file edited while I only worked on the **workflow runs**. I am not able to pinpoint the why. But I guess it may have something to do with the **lint formatting** in some instances as well the _gitignore_ having file entry after they are staged.